intranet/apps/bus/tasks.py

```diff
@@ -12,3 +17,54 @@ def reset_routes() -> None:
 
     for route in Route.objects.all():
         route.reset_status()
+
+
+@shared_task
+def fetch_fcps_bus_delays():
+    now = timezone.localtime(timezone.now())
+    # Check if the current time is within 6-9 AM or 3-6 PM from Mon-Fri
+    if now.weekday() in (5, 6):
+        return
+    if not (now.hour in range(6, 9) and now.hour in range(15, 18)):
```

Comment:
This if statement will always trigger because the current hour can never be in `[6, 9)` AND `[15, 18)` at the same time
```suggestion
    if not (now.hour in range(6, 9) or now.hour in range(15, 18)):
```